### ISSUE TITLE
grass: Fails to build with "MySQL includes directory [...] does not exist." on NixOS 19.09

### ISSUE THREAD

**Issue by das-g** [NixOS/nixpkgs#72772]
**Describe the bug**
GRASS GIS fails to build on NixOS 19.09

**To Reproduce**
1. `nix build nixpkgs.grass`

**Expected behavior**
The command builds GRASS and terminates successfully. (Exit code `0`)

**Observed behavior**
The command outputs
```
builder for '/nix/store/v8x81d41ym8c0yvb7p4i32rgznggh1ax-grass.drv' failed with exit code 1; last 10 log lines:
  checking for png_read_image in -lpng... yes
  checking whether to use PostgreSQL... yes
  checking for location of PostgreSQL includes... 
  checking for libpq-fe.h... yes
  checking for location of PostgreSQL library... /nix/store/1ra20pv1ywis5q9s1n31r2nivz07chh5-postgresql-9.6.15-lib/lib/
  checking for PQsetdbLogin in -lpq... yes
  checking for PQcmdTuples in -lpq... yes
  checking whether to use MySQL... yes
  checking for location of MySQL includes... /nix/store/wpwcramih0y4f4bip2j8fpi33n9kim94-mariadb-connector-c-2.3.7/include/mysql
  configure: error: *** MySQL includes directory /nix/store/wpwcramih0y4f4bip2j8fpi33n9kim94-mariadb-connector-c-2.3.7/include/mysql does not exist.
[0 built (1 failed)]
error: build of '/nix/store/v8x81d41ym8c0yvb7p4i32rgznggh1ax-grass.drv' failed
```

Exit code is `130`.

**Screenshots**
![Bildschirmfoto von 2019-11-04 11-40-47](https://user-images.githubusercontent.com/97746/68115134-f8705780-fef7-11e9-8f0e-554073aae57e.png)

**Additional context**

n/a

**Metadata**
Result of running `nix run nixpkgs.nix-info -c nix-info -m`:

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.81, NixOS, 19.09.1019.c5aabb0d603 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"nixos-19.09.1019.c5aabb0d603"`
 - channels(das-g): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
  - "grass"
# a list of nixos modules affected by the problem
module:
  # How do I find out?
```

**Comment by aanderse** [comment]
cc @Izorkin

**Comment by Izorkin** [comment]
cc @ttuegel 
After this PR - https://github.com/NixOS/nixpkgs/pull/70025 error build grass package.

**Comment by bennofs** [comment]
Grass now builds successfully both on master and in release-19.09. There is still an error when running it, but that doesn't seem to be related to the build error?

**Comment by das-g** [comment]
The build problems seem indeed to be fixed in both NixOS 20.03 and NixOS unstable.

```bash
nix build -f channel:nixos-unstable grass
```
and
```bash
nix build -f channel:nixos-20.03 grass
```
both succeed.

Closing this issue. Let's file a separate one for the runtime errors.

**Comment by das-g** [comment]
> Let's file a separate one for the runtime errors.

Done:  #93664

> Grass now builds successfully both on master and in release-19.09. There is still an error when running it, but that doesn't seem to be related to the build error?

@bennofs Are/were you seeing the same error as described in #93664? If not, please file a separate issue.
